app/Models/User.php

```diff
@@ -102,8 +101,22 @@ static public function engineerList()
       if ($user->legacy()) {
         continue;
       }
+      // Filter out inactive users
+      if ($user->inactive === 1) {
```

Comment:
Is there a reason to not just check the falsey value (`$user->inactive`) and get rid of `=== 1`? Or does this work weirdly in php?

Reply:
i tested it and it seems to work. i wanted to be extra careful because of php but in this case the only two values possible are 0 and 1 (no nulls) so i think it's ok